test/testSamples/MaterialX/DemoQuads.usda

```diff
@@ -580,6 +566,68 @@ def Scope "mtl"
             float2 outputs:out
         }
     }
+
+    def Material "standard_surface1"
+    {
+        token outputs:mtlx:surface.connect = </mtl/standard_surface1/standard_surface1.outputs:out>
+
+        def Shader "standard_surface1"
+        {
+            uniform token info:id = "ND_standard_surface_surfaceshader"
+            float3 inputs:normal.connect = </mtl/standard_surface1/normalmap1.outputs:out>
+            token outputs:out
+        }
+
+        def Shader "normalmap1"
+        {
+            uniform token info:id = "ND_normalmap"
+            float3 inputs:normal.connect = </mtl/standard_surface1/image1.outputs:out>
+            float3 outputs:out
+        }
+
+        def Shader "image1"
+        {
+            uniform token info:id = "ND_image_vector3"
+            asset inputs:file = @textures/mesh_wire_norm.png@
+            string inputs:filtertype = "closest"
+            float3 outputs:out
+        }
+    }
+
+    def Material "standard_surface2"
+    {
+        token outputs:mtlx:surface.connect = </mtl/standard_surface2/standard_surface1.outputs:out>
+
+        def Shader "standard_surface1"
+        {
+            uniform token info:id = "ND_standard_surface_surfaceshader"
+            float3 inputs:normal.connect = </mtl/standard_surface2/normalmap1.outputs:out>
+            token outputs:out
+        }
+
+        def Shader "normalmap1"
+        {
+            uniform token info:id = "ND_normalmap"
+            float3 inputs:in.connect = </mtl/standard_surface2/image1.outputs:out>
+            float3 outputs:out
+        }
+
+        def Shader "image1"
+        {
+            uniform token info:id = "ND_image_vector3"
+            asset inputs:file = @textures/mesh_wire_norm.png@
+            string inputs:filtertype = "closest"
+            float2 inputs:texcoord.connect = </mtl/standard_surface2/geompropvalue1.outputs:out>
+            float3 outputs:out
+        }
+
+        def Shader "geompropvalue1"
+        {
+            uniform token info:id = "ND_geompropvalue_vector2"
+            string inputs:geomprop = "st1"
```

Comment:
In this case, since this is the geompropreader connected to the normalmap image, this will be also connected to the auto-added texcoordtangents node.